src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -245,22 +178,43 @@ public function editAction($featureId, Request $request)
         ]);
     }
 
-    public function getHelperListBridge(): HelperListFeatureBridge
+    /**
+     * @return ControllerConfiguration
+     */
+    public function getControllerConfiguration(): ControllerConfiguration
+    {
+        return $this->buildControllerConfiguration(
+            'feature',
+            Feature::class,
+            'AdminFeatures'
+        );
+    }
+
+    /**
+     * @return FeatureHelperListBridge
+     */
+    public function getHelperListBridge(): FeatureHelperListBridge
```

Comment:
This method is related to the list interface right? So if we decide to remove the interface we can remove this one as well (or at least turn it into a protected method)

The only use I can see for now about the interface is that it forces the user to implement this method, so it's kind of a guide to help you build the controller. But if it's only that it's quite limited, this method alone only does half (or even less) of the job, however it forces a public method which is actually never called from outside. So the gain is quite limited.